app/views/notifications/_notification.html.erb

```diff
@@ -1,10 +1,10 @@
-<%# locals: (profile:, message:) %>
+<%# locals: (notification:, speaker: notification.record.speakers&.first) %>
 
```

Comment:
It would be nice if users could click on a notification and be redirected to the session#show
![image](https://github.com/user-attachments/assets/c95b2796-e297-4913-88f9-6c0576902aa0)